npm_and_yarn/helpers/lib/pnpm/lockfile-parser.js

```diff
@@ -15,6 +15,7 @@ async function parse(directory) {
   });
 
   return Object.entries(lockfile.packages ?? {})
+    .filter(([depPath, pkgSnapshot]) => dependencyPath.parse(depPath).name) // exclude empty names
```

Comment:
Thought: can `dependencyPath.parse()` return null/undefined and do we need to handle that? Answer: [no](https://github.com/pnpm/pnpm/blob/43bd75c5b82e1c61c579eecccc0a75acf18732d9/packages/dependency-path/src/index.ts#L91-L129) 👍

Reply:
Answer yes, I am updating the code now.

Reply:
@bdragon Thanks and updated.

Reply:
Sorry if my comment wasn't clear. I was just checking to see if this was something we needed to handle, but what you had was fine so there was no need to change it.